app/lib/pages/home_page.dart

```diff
@@ -20,17 +22,20 @@ class HomePage extends StatefulWidget {
 class _HomePageState extends State<HomePage> {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(widget.title),
-        centerTitle: true,
-        elevation: 0.0,
-      ),
-      body: PostsList(_loadPosts(context)),
-      floatingActionButton: SignInFab(
-        auth: Auth(
-          firebaseAuth: FirebaseAuth.instance,
-          googleSignIn: GoogleSignIn(),
+    return AnnotatedRegion<SystemUiOverlayStyle>(
```

Comment:
Please add a single expect to the `Renders list of posts` test example that `AnnotatedRegion` exists in the tree.

Reply:
- [x] Test added in 4287a23, inspired by [official flutter AnnotatedRegion test](https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/annotated_region_test.dart#L10).